MAUI/Maps/getting-started.md

```diff
@@ -267,6 +267,42 @@ public MainPage()
 
 ![maps basic view](images/getting-started/map_basic_view.png)
 
+## Custom shapes
+
+The MAUI Maps control allows you to render any custom shape to make a map look like building infrastructure, a sports stadium, plane or bus seat arrangements, and more using `Geometry` property.
+
+{% tabs %}
+
+{% highlight XAML %}
+
+<map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapShapeLayer ShapesSource="https://cdn.syncfusion.com/maps/map-data/seating.json" 
+                           Geometry="Points" />
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+public MainPage()
+{
+    InitializeComponent();
+    SfMaps map = new SfMaps();
+    MapShapeLayer layer = new MapShapeLayer();
+    layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/seating.json"));
+    layer.Geometry = MapGeometryType.Points;
+    map.Layer = layer;
+    this.Content = map;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![maps basic view](images/getting-started/maps_cartesian_view.png)
```

Comment:
Currently we don't have multiple selection support. So, please change the output image without selection.